tempesta_fw/http_parser.c

```diff
@@ -1844,8 +1921,8 @@ __req_parse_accept(TfwHttpReq *req, unsigned char *data, size_t len)
 			__FSM_I_MOVE(I_EoT);
```

Comment:
It seems we allow `Accept: text/html; q=0.5, , ,,,, text/plain`. Please add the case to the tests. And this is allowed case, see https://tools.ietf.org/html/rfc7230#section-7 .